test/Dispatcher/DispatcherTest.php

```diff
@@ -479,10 +498,10 @@ public function provideMethodNotAllowedDispatchCases() {
         $callback = function(RouteCollector $r) {
             $r->addRoute(['GET', 'POST'], '/user', 'handlerGetPost');
             $r->addRoute(['DELETE'], '/user', 'handlerDelete');
-            $r->addRoute([], '/user', 'handlerNone');
+            $r->addRoute("PATCH", '/user', 'handlerPatch');
```

Comment:
my bad, is not consistent with the style of the file, will change later…

In general, I follow the inverse philosophy … double quotes unless single quotes necessary.